components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -245,9 +245,14 @@ IN_PROC_BROWSER_TEST_F(BraveRewardsBrowserTest, ToggleAutoContribute) {
     "    count -= 1;"
     "  }"
     "  if (document.querySelector(\"[data-test-id2='autoContribution']\")) {"
-    "    clearInterval(interval);"
-    "    resolve(document.querySelector(\"[data-test-id2='autoContribution']\")"
-    "      .getAttribute(\"data-toggled\") === 'false');"
+    "    if (!toggleClicked) {"
```

Comment:
I'm having a little trouble understanding the fix, can you explain why it didn't work before? Did you test that it fails if you intentionally break the functionality in the code?

Reply:
I'm not sure why it was even working on some platforms. For this fix on ac table toggle. The toggle wasn't available right away so we do a page reload. It seems the `click` event was being tried before it was completely available. With this, we postpone the click event until we have detected that we have the element.